faqs.md

```diff
@@ -21,13 +21,20 @@ If there’s already an [existing proposal for grants](https://github.com/aragon
 - Make sure the [Proposals for grants](https://github.com/aragon/nest/issues) doesn’t have an existing proposals for something similar to your proposal. If there is an existing one that is close to your idea, join in the discussion of that Proposal to see if yours could be integrated into the existing one, or if you should create a new proposal
 - If nothing similar exists, create [a new Proposal for grants](https://github.com/aragon/nest/issues/new) at the [Nest Repository](https://github.com/aragon/nest/) and fill it out in detail following the [Guide for submitting a proposal for grants](guides/guide_for_submitting_a_proposal_for_grants.md)
 
-## **Who will be deciding on how the grants are distributed?**
-The funding will be described in the Proposal of the grant. The total amount, how many portions will it be divided into, timetable and milestones will be up for discussion.
+## **How does the application process works?**
+The application process has two stages:
+
+1. The proposal stage (see above). The main focus in this stage is the discussion about the idea/project and whether it should be considered for a grant. Once the proposal is approved we move to the second stage. 
+
+2. The request for funding (rff) stage (see above). The main focus in this stage is the discussion about the team, timeline/roadmap, milestones and deliverables. Here we take the final decision on whether to fund the team.
 
-Placeholder and Aragon are the ones making the **initial** decisions. But as soon as the [Aragon Voting App](../dev/apps/voting.md) and it’s accompanying signaling mechanism is ready, **the community will have the final decision** on which proposals are granted funding.
+There are no deadlines for applications. The whole application process can take from 2 weeks up to a month but the length of the process depends on a case by case basis. 
+
+## **Who will be deciding on how the grants are distributed?**
+The funding will be described in the proposal and rff stages of the grant. The total amount, how many portions will it be divided into, timetable and milestones will be up for discussion with the relevant team.
```

Comment:
For clarity, let's just write out "request for funding" instead of the abbreviation.

Reply:
Ok!